### ISSUE TITLE
mssql ALTER TABLE

### ISSUE THREAD

**Issue by kPlusPlus** [holistics/dbml#418]
Dear Guru dbml
problem is
convert from sql to dbml
no convert any like thi

ALTER TABLE [dbo].[tUD]  WITH CHECK ADD  CONSTRAINT [FK_tUD_tU] FOREIGN KEY([u_id])
REFERENCES [dbo].[tU] ([id])
ON DELETE CASCADE
GO

**Comment by TeaNguyen** [comment]
Yes, you are correct. We do not currently support or parse ALTER statements. However, we are gradually improving that ability.

With https://github.com/holistics/dbml/pull/416 (we'll try to revamp the PostgreSQL parser 1st), we're first attempting to redesign our SQL parsers to reduce parsing errors and support almost all SQL syntax. Then we'll try to cover these ALTER statements to generate the most accurate DBML of the database's current state.
